Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -78,14 +78,15 @@ def runSelected(opt):
                     250202.181, # RelValTTbar_13                PREMIX   
 
                     # Run3
-                    11634.0,    # TTbar_14TeV 
-                    13234.0,    # RelValTTbar_14TeV             FastsSim
-                    12434.0,    # RelValTTbar_14TeV
-                    12446.0,    # RelValZEE_13
-                    12634.0,    # RelValTTbar_14TeV             PU = Run3_Flat55To75_PoissonOOTPU
-                    12434.7,    # RelValTTbar_14TeV             mkFit
+                    11634.0,    # TTbar_14TeV                   2021
+                    13234.0,    # RelValTTbar_14TeV             2021 FastsSim
+                    12434.0,    # RelValTTbar_14TeV             2023
```

Comment:
any strong reason to keep this in the system?

Reply:
OK, just for the record, this is what is causing the changes explained at https://github.com/cms-sw/cmssw/pull/44026#issuecomment-1963539080.

Reply:
Hi,
I think we should keep at least one workflow around, to monitor its healthy if we will have legacy wf later (not hlt part, of course).

Reply:
> I think we should keep at least one workflow around, to monitor its healthy if we will have legacy wf later

not proposing to remove entirely btw, just in the limited matrix.